src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const costs = 40;
+
+  if (days < 3) {
+    result = days * costs;
+  } else if (days < 7) {
+    result = days * costs - 20;
+  } else {
+    result = days * costs - 50;
+  }
```

Comment:
avoid such nested `if else` conditions
This code is hard to read and maintain

Reply:
and what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean